metals/src/main/scala/scala/meta/internal/bsp/BspServers.scala

```diff
@@ -94,15 +94,19 @@ final class BspServers(
             arg
         }
 
+      val additionalEnv = JdkSources
+        .defaultJavaHome(userConfig().javaHome)
+        .map("JAVA_HOME" -> _.toString()) :+
+        /* let's make sure that PWD is set correctly, especially important for Bazel
+         */
+        ("PWD" -> projectDirectory.toString())
```

Comment:
I wonder if we shouldn't do that in a more generic way at set this env var in `SystemProcess` instead.

Reply:
Moved there, should probably be better.